dd-sdk-android/src/test/kotlin/com/datadog/android/instrumentation/gestures/NoOpWindowCallbackTest.kt

```diff
@@ -0,0 +1,261 @@
+package com.datadog.android.instrumentation.gestures
+
+import android.view.ActionMode
+import android.view.KeyEvent
+import android.view.Menu
+import android.view.MenuItem
+import android.view.MotionEvent
+import android.view.SearchEvent
+import android.view.View
+import android.view.Window
+import android.view.WindowManager
+import android.view.accessibility.AccessibilityEvent
+import com.datadog.android.instrumentation.gesture.NoOpWindowCallback
+import com.datadog.android.utils.forge.Configurator
+import com.nhaarman.mockitokotlin2.mock
+import com.nhaarman.mockitokotlin2.verifyZeroInteractions
+import fr.xgouchet.elmyr.Forge
+import fr.xgouchet.elmyr.junit5.ForgeConfiguration
+import fr.xgouchet.elmyr.junit5.ForgeExtension
+import org.junit.jupiter.api.BeforeEach
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.api.extension.ExtendWith
+import org.junit.jupiter.api.extension.Extensions
+import org.mockito.Mock
+import org.mockito.junit.jupiter.MockitoExtension
+import org.mockito.junit.jupiter.MockitoSettings
+import org.mockito.quality.Strictness
+
+@Extensions(
+    ExtendWith(
+        MockitoExtension::class,
+        ForgeExtension::class
+    )
+)
+@ForgeConfiguration(Configurator::class)
+@MockitoSettings(strictness = Strictness.LENIENT)
+internal class NoOpWindowCallbackTest {
```

Comment:
I appreciate the feeling, but I'm not sure NoOp implementation should be tested. Especially since you don't test anything here, merely that no exception is thrown.

Reply:
I agree....it is a PITA to test those in any case